styleguide/source/_patterns/02-molecules/tools-learn-more.twig

```diff
@@ -6,7 +6,7 @@
   <ul>
     {% block learMoreLinks %}
       {% for link in learnMore.links %}
-        {% set link = link %}
+        {% set link = link|merge({ "class": "ama__link--black" }) %}
```

Comment:
Setting the class on the link isn't going to change the implementation on Drupal. We should extend the class into the CSS for that link instead of having to manually add the class in all places.

Reply:
This would hold true of h tags throughout the style guide. Are you saying we should use SCSS extends for the h tags too?

Reply:
We can then say @extend h1 for the page title instead of heading { "level": "1" }

Reply:
I do feel that way, but possibly this is something to discuss during the FED meeting. I've added this PR to the agenda.